File: swagger/models/get_gift_card_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .gift_card import GiftCard

class GetGiftCardResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		gift_cards
		Contains information about the gift cards.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'GiftCards': ['gift_cards', GiftCard],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'gift_cards': 'GiftCards',
		}

	pagination_response = None
	gift_cards = None

File: swagger/Models/GetGiftCardResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetGiftCardResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the gift cards.
	 * 
	 * @var GiftCard[] 
	 */
	public $giftCards;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'GiftCards' => ['giftCards', 'array', GiftCard::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'giftCards' => ['GiftCards', 'array', GiftCard::class],
			];
	}
}
